packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -6711,6 +6793,50 @@ addEventListener('fetch', event => {});`
 			        For more details, see [4mhttps://developers.cloudflare.com/workers/cli-wrangler/configuration[0m
 
 			"
+		`);
+			});
+			it("should error if using unsafe bindings capnp_schemas which are invalid", async () => {
```

Comment:
Add a blank line between tests.

```suggestion

			it("should error if using unsafe bindings capnp_schemas which are invalid", async () => {
```